### DISCUSSION TITLE
note-linear-algebra-a1/3/

### DISCUSSION THREAD

**Discussion by giscus** [Cgfyufsygsm/cgfyufsygsm.github.io#65] [Announcements]
# note-linear-algebra-a1/3/

向量组的极大无关组与秩  极大无关组的定义：KnK^nKn 中一个向量组的一个部分组称为一个极大线性无关组，若该部分组本身线性无关，但是从原向量组的其余向量中任选一个添进去，得到的新部分组都线性相关。 向量组等价的定义：若 α1,⋯ ,αs\alpha_1, \cdots ,\alpha_sα1​,⋯,αs​ 与 β1,⋯ ,βr\beta_1, \cdots ,\beta_rβ1​,⋯,βr​

https://blog.imyangty.com/note-linear-algebra-a1/3/

**Comment by PsephurusGladius** [comment]
感觉用 Steinitz 替换定理引出线性无关的一堆性质会比较美妙
